src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  font-family: Roboto, Aria, sans-serif;
+}
+
+:root {
+  --header-active-link-color: #00acdc;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 10px 50px
+}
+
+.page-wrap {
+  display: flex;
+}
+
+.logo {
+  flex: 1;
+}
+
+.menu {
+   margin-top: -10px;
+}
+
+.list {
+  display: flex;
+  height: 60px;
+  flex: none;
+}
+
+.list li {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  margin: 0 10px;
+}
+
+.list li:last-child {
+  margin-right: 0;
+}
+
+.list li:first-child {
+  margin-left: 0;
+}
+
+.list li a {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  text-transform: uppercase;
+  height: 100%;
+  font-weight: 500;
+}
+
+.nav-links .list [data-qa="hover"]:hover {
+  color: var(--header-active-link-color);
+}
```

Comment:
you should change color for all links
data-qa="hover" you need for tests, test will search item with such attribute and will hover it.